### ISSUE TITLE
Arduiono IDE upload not working any more

### ISSUE THREAD

**Issue by rmh78** [nasys/badgerboard#9]
I developed my arduino program and uploaded it all the time without problems.
After one upload it did not work any more. 

The arduino IDE showed this error message:
avrdude: butterfly_recv(): programmer is not responding
Found programmer: Id = "Configu"; type = A
    Software Version = .�; Hardware Version = O.T
avrdude: error: buffered memory access not supported. Maybe it isn't
a butterfly/AVR109 but a AVR910 device?
avrdude: initialization failed, rc=-1
         Double check connections and try again, or use -F to override
         this check.

avrdude: error: programmer did not respond to command: leave prog mode
avrdude: error: programmer did not respond to command: exit bootloader

Do you have any idea what happened to my badgerboard?

**Comment by rmh78** [comment]
If I open the seriell monitor I see that my last uploaded program is still running.
But I'm not able to upload any other program. I also tried the simple blick example (same error).

**Comment by rmh78** [comment]
Like @patmolloy wrote in his issue I read the article "Installing an Arduino Bootloader" on https://learn.sparkfun.com/tutorials/installing-an-arduino-bootloader and gave it a try.

1) I used my Arduino UNO as a programmer and uploaded the sketch ArduinoISP to it. 
2) I connected the Badgerboard to the Arduino UNO as described in the article
3) In the Arduino IDE I choose as Board "LilyPad Arduino USB" and called "Burn Bootloader"

And everything worked fine and I'm able to upload sketches again :-)

Thanks to @patmolloy for this hint!
